Networking/Networking/Model/Product/CookieJar.swift

```diff
@@ -0,0 +1,164 @@
+import Foundation
+import WebKit
+
+/// Provides a common interface to look for a logged-in WordPress cookie in different
+/// cookie storage systems, to aid with the transition from UIWebView to WebKit.
+///
+@objc protocol CookieJar {
```

Comment:
Suggestion!

If none of this code is gonna be used / consumed in ObjC, you can drop the @objc modifier (everywhere). Same for NSObject as superclass!